var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -33,6 +33,7 @@ class Scotch(Package):
     url      = "http://gforge.inria.fr/frs/download.php/latestfile/298/scotch_6.0.4.tar.gz"
     list_url = "http://gforge.inria.fr/frs/?group_id=248"
 
+    version('6.0.5a', '8430dff7175a1dfd5a3258e75260cf71')
```

Comment:
Is this an alpha release? We don't normally add alpha releases to Spack, as they tend to be very unstable. At the very least, if you want to add an alpha release, you should mark the previous version as `preferred=True`.

Reply:
Strange. Their downloads page also lists many previous versions with `a` and `b` in them:
https://gforge.inria.fr/frs/?group_id=248

Perhaps it was just a small patch fix of 6.0.5 that was released soon after.